pkg/kubelet/sysctl/safe_sysctls.go

```diff
@@ -51,19 +51,29 @@ var safeSysctlsIncludeReservedPorts = []string{
 // - it is namespaced in the container or the pod
 // - it is isolated, i.e. has no influence on any other pod on the same node.
 func SafeSysctlAllowlist() []string {
-	if goruntime.GOOS == "linux" {
-		// make sure we're on a new enough kernel that the ip_local_reserved_ports sysctl is namespaced
-		kernelVersion, err := getKernelVersion()
-		if err != nil {
-			klog.ErrorS(err, "Failed to get kernel version, dropping net.ipv4.ip_local_reserved_ports from safe sysctl list")
-			return safeSysctls
-		}
-		if kernelVersion.LessThan(version.MustParseGeneric(ipLocalReservedPortsMinNamespacedKernelVersion)) {
-			klog.ErrorS(nil, "Kernel version is too old, dropping net.ipv4.ip_local_reserved_ports from safe sysctl list", "kernelVersion", kernelVersion)
-			return safeSysctls
-		}
+	if goruntime.GOOS != "linux" {
+		return append(safeSysctls, sysctlIpLocalReserverdPorts, sysctlTcpKeepAliveTime)
 	}
-	return safeSysctlsIncludeReservedPorts
+
+	kernelVersion, err := getKernelVersion()
+	if err != nil {
+		klog.ErrorS(err, "Failed to get kernel version, dropping net.ipv4.ip_local_reserved_ports and net.ipv4.tcp_keepalive_time from safe sysctl list")
+		return safeSysctls
```

Comment:
We don't want to have to update the error message every time a new sysctl is added. Just say "unable to determine which sysctls are available" or something like that. (This error shouldn't ever even happen anyway.)